### ISSUE TITLE
Weather card

### ISSUE THREAD

**Issue by bastshoes** [home-assistant/home-assistant-polymer#2297]
**Home Assistant release with the issue:**
0.84.1

**Last working Home Assistant release (if known):**
0.83.3

**Browser and Operating System:**

Chrome, Firefox
**Description of problem:**

I have a view with weather card for Yahoo weather component.

It's not shown in view and browser through error in console.

**Javascript errors shown in the web inspector (if applicable):**
```
TypeError: this.config is undefined[Learn More] bf7146608f595468a214.chunk.js:3157 
computeName
bf7146608f595468a214.chunk.js:3157
runMethodEffect
app-0d042052.js:248
_evaluateBinding
app-0d042052.js:248
runBindingEffect
app-0d042052.js:248
runEffectsForProperty
app-0d042052.js:248
runEffects
app-0d042052.js:248
_propagatePropertyChanges
app-0d042052.js:248
_propertiesChanged
app-0d042052.js:248
_flushProperties
app-0d042052.js:895
_flushProperties
app-0d042052.js:248
ready
app-0d042052.js:248
ready
app-0d042052.js:232
ready
bf7146608f595468a214.chunk.js:3157
_enableProperties
app-0d042052.js:895
connectedCallback
app-0d042052.js:224
connectedCallback
app-0d042052.js:232
dynamicContentUpdater
app-0d042052.js:2637
_updateCard
bf7146608f595468a214.chunk.js:3388
createObserver/this.observer<
bf7146608f595468a214.chunk.js:3388
```

**Additional information:**

In addition error in HASS log recorded:
```
https://hass.homelab.loc/frontend_latest/bf7146608f595468a214.chunk.js:3157:1358 TypeError: this.config is undefined
4:52 PM components/system_log/__init__.py (ERROR)
```

**Comment by iantrich** [comment]
@bastshoes Lovelace card? Please share config

**Comment by iantrich** [comment]
Please try a different weather provider. This might be related to yahoo alone.

**Comment by AxelRHD** [comment]
> Please try a different weather provider. This might be related to yahoo alone.

I also get an error with darksky (as stated in issue #2294).

**Comment by Mariusthvdb** [comment]
none of the regular weather components show in the regular (now legacy) HA. They do show in Lovelace.

using:

weather:
  - platform: yweather
    name: Home Yh

  - platform: openweathermap
    api_key: !secret openweather_key
    name: Home Owm
    mode: daily

  - platform: buienradar
    name: !secret buienradar_station

  - platform: darksky
    api_key: !secret darksky_key
    name: Home Darksky
    mode: daily

they are in dev-state:

![schermafbeelding 2018-12-13 om 23 30 26](https://user-images.githubusercontent.com/33354141/49973101-1590e600-ff34-11e8-9930-b7b7cbe11371.jpg)

not in frontend.

Lovelace shows them fine

**Comment by Ryssk** [comment]
I can confirm that NONE of the weather components or cards work with the default HassIO ui. Atleast not in Frontend!

But they do work in LovelaceUI

**Comment by m0wlheld** [comment]
As mentioned in #2310, using attributes of (at least dark sky) weather component e.g. as template sensor works fine. See #2310 for details.

**Comment by SamuiSmartHome** [comment]
I got the weather card show on both classic ui (yweather) and lovelace (darksky). But all switch, sensor, binary_sensor and more all dissapear from frontend classic ui. the entity_id still see and show correct status.

**Comment by m0wlheld** [comment]
Confirm fixed with Home Assistant 0.84.3
